website/pages/docs/k8s/upgrade/index.mdx

```diff
@@ -7,63 +7,303 @@ description: Upgrade Consul on Kubernetes
+
+   **Before upgrading, be sure you've read the other sections on this page.**
+
+~> NOTE: It's important to always set the `--version` flag because otherwise Helm
+will use the most up-to-date version in its local cache which may result in an
+unintended upgrade.
+
+### Helm Chart Version Upgrade
+
+You may wish to upgrade your Helm chart version to take advantage of new features,
+bugfixes, or because you want to upgrade your Consul version and it requires a
+certain Helm chart version.
+
+1. Update your local Helm repository cache:
+
+```bash
+helm repo update
+```
+
+1. You can then list all available versions:
+
+```bash
+helm search repo hashicorp/consul -l
+NAME            	CHART VERSION	APP VERSION	DESCRIPTION
+hashicorp/consul	0.24.1       	1.8.2      	Official HashiCorp Consul Chart
+hashicorp/consul	0.24.0       	1.8.1      	Official HashiCorp Consul Chart
+...
+```
+
+Here we can see that the latest version of `0.24.1`.
+
+1. To determine which version you have installed, run:
+
+```bash
+helm list -f consul
+NAME  	NAMESPACE	REVISION	UPDATED        	STATUS  	CHART        	APP VERSION
+consul	default  	2       	2020-09-30 ...	deployed	consul-0.24.0	1.8.2
+```
+
+Here we can see we're running version `0.24.0` (from `consul-0.24.0`) and so
+we may want to upgrade to the latest `0.24.1` version.
+
+1. Read the changelog for that version and any versions in between: https://github.com/hashicorp/consul-helm/blob/master/CHANGELOG.md.
+
+1. Upgrade by running `helm upgrade` with the `--version` flag:
+
+```bash
+helm upgrade consul hashicorp/consul --version 0.24.1 -f /path/to/my/values.yaml
+```
+
+**Before upgrading, be sure you've read the other sections on this page.**
+
+### Consul Version Upgrade
+
+If a new version of Consul is released, you will need to run a Helm upgrade
+to update to the new version.
+
+1. Ensure you've read the [Upgrading Consul](/docs/upgrading) documentation.
+1. Ensure you've read any [specific instructions](/docs/upgrading/upgrade-specific) for the version you're upgrading
+   to and the Consul [changelog](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md) for that version.
+1. Read our [Compatibility Matrix](/docs/k8s/upgrade/compatibility) to ensure
+   your current Helm chart version supports this Consul version. If it does not,
+   you may need to also upgrade your Helm chart version at the same time.
+1. Set `global.consul` in your `values.yaml` to the desired version:
+   ```yaml
+   global:
+     image: consul:1.8.3
+   ```
+1. Determine your current installed chart version
+
+```bash
+helm list -f consul
+NAME  	NAMESPACE	REVISION	UPDATED        	STATUS  	CHART        	APP VERSION
+consul	default  	2       	2020-09-30 ...	deployed	consul-0.24.0	1.8.2
+```
+
+In this example, I'm running version `0.24.0` (from `consul-0.24.0`).
+
+1. Run `helm upgrade consul hashicorp/consul --version 0.24.0 -f /path/to/my/values.yaml`.
```

Comment:
```suggestion
1. Perform a `helm upgrade`: 

```bash
helm upgrade consul hashicorp/consul --version 0.24.0 -f /path/to/my/values.yaml
```

```

Reply:
Same code fencing issue here.